app/src/main/java/com/firebase/uidemo/auth/AuthUiActivity.java

```diff
@@ -34,7 +34,7 @@
 import com.firebase.ui.auth.AuthUI;
 import com.firebase.ui.auth.AuthUI.IdpConfig;
 import com.firebase.ui.auth.IdpResponse;
-import com.firebase.ui.auth.ui.ResultCodes;
+import com.firebase.ui.auth.ResultCodes;
```

Comment:
We can remove deprecated things after a major version change (if we want to).  Or we can leave it for life.  This will be the first example of deprecation in this library so there's no precedent for us, and every library does it a bit differently.